CHANGELOG.md

```diff
@@ -7,6 +7,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.4.0] - October, 10, 2022
+
+### React Components
+
+#### Added
+
+- [SSR Support] for hds-react components
+- [Icons] A total of 13 new UI icons added
+
+#### Fixed
+
+- [CookieConsent] Cookie consent title is now refocused on language change and the banner won't shrink unexpectedly
+- [Link] Ability to pass an aria-label added to the Link component
+- [TimeInput] Fixed an accessibility bug by grouping hours and minutes inputs under the same label. Added group="role" to the parent div of TimeInput component's hours and minutes inputs.
```

Comment:
- [TimeInput] Fixed an accessibility bug by using the correct id with the helper texts. This way the screen readers can connect the helper text to inputs.